docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -37,128 +37,121 @@ To find the `volumeHandle`, use one of the following procedures:
+     The Properties window appears. Use the UID number.
 
-      For more information about Spectrum Virtualize products, find your product information in [IBM Documentation](https://www.ibm.com/docs/).
+     For more information about Spectrum Virtualize products, find your product information in [IBM Documentation](https://www.ibm.com/docs/).
 
-- Through the IBM Hyper-Scale Manager user interface for FlashSystem A9000 and A90000R storage systems:
+- **Through the IBM Hyper-Scale Manager user interface for FlashSystem A9000 and A9000R storage systems:**
 
-  1.  Select **Pools and Volumes Views** > **Volumes** from the side bar.
+  1. Select **Pools and Volumes Views** > **Volumes** from the side bar.
 
       The **Volumes** table is displayed.
 
-  2.  Select the `Volume`.
+  2. Select the `Volume`.
 
       The **Volume Properties** form appears.
 
-  3.  Use the **ID** number.
+  3. Use the **ID** number.
     
       For more information, see [IBM Hyper-Scale Manager documentation](https://www.ibm.com/docs/en/hyper-scale-manager/).
 
 
-Use this procedure to help build a PV yaml file for your volumes.
+Use this procedure to help build a PV YAML file for your volumes.
 
 **Note:** These steps are setup for importing volumes from a Spectrum Virtualize Family system. Change parameters, as needed.
 
-1.  Create a persistent volume (PV) yaml file.
+1. Create a persistent volume (PV) YAML file.
 
     **Important:** Be sure to include the `storageClassName` and `controllerPublishSecretRef` parameters or errors will occur.
 
-2.  Take the `volume_name` and other optional information (collected before the procedure) and insert it into the yaml file.
-
-    <pre>
-    apiVersion: v1
-    kind: PersistentVolume
-    metadata:
-      # annotations:
-        # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
-      name: vol1-pv
-    spec:
-      accessModes:
-      - ReadWriteOnce
-      capacity:
-        storage: 1Gi
-      csi:
-        controllerPublishSecretRef:
-          name: demo-secret
-          namespace: default
-        driver: block.csi.ibm.com
-        <b># volumeAttributes:
-          # pool_name: ibmc-block-gold
-          # storage_type: SVC
-          # volume_name: vol1
-          # array_address: baremetal10-cluster.xiv.ibm.com
-        volumeHandle: SVC:600507640082000B08000000000004FF</b>
-      # persistentVolumeReclaimPolicy: Retain
-      storageClassName: ibmc-block-gold
-      # volumeMode: Filesystem
-    </pre>
-
-3.  Create a PersistentVolumeClaim (PVC) yaml file.
+2. Take the `volume_name` and other optional information (collected before the procedure) and insert it into the YAML file (under `spec.csi.volumeAttributes`).
+
+    **Important:** If using the CSI Topology feature, the `spec.csi.volumeHandle` contains the system ID. In the example below, the `spec.csi.volumeHandle` would read similar to the following: `SVC:demo-system-id-1:600507640082000B08000000000004FF`.
```

Comment:
@ArbelNathan should we add the vdisk id; here?

Reply:
I think we should, so Rivka could add a replication limitation, that goes something like:
```
Volumes created by a version of the driver older than 1.7.0, need to be imported using the version of the import procedure no older than 1.7.0, before they could be replicated
```

Reply:
1. @oriyarde  are you saying that we should add an additional parameter (vdisk id) that they need to find and insert into the yaml? - If so, please let me know how to find it in all of the cases that we already write about. (CLI for SVC/ CLI for A9K / SVC GUI / HSM)
2. regarding the limitation - I wouldn't put it as a limitation, specifically, but more as just an important note at the beginning of the section (IMO). but are you saying that just if they're importing from an earlier version of csi driver they need to use this specific version document? I don't know that that is necessary to put in.

Reply:
1. yes (though "vdisk" is an SVC-only term, I would call it "internal id"), the commands for it are the same as the existing commands for the volumeHandle
2. which section? this backward incompatibility issue is only relevant for replication, so IMO it doesn't belong in this page. it's not about `if` they're importing, it's the fact that they `need` to import with 1.7.0+ import procedure in order to use replication (after they removed the old k8s volume object using "Retain" [`deletionPolicy`](https://kubernetes.io/docs/concepts/storage/volume-snapshot-classes/#deletionpolicy)).

Reply:
So you suggest to add this as a replication limitation? - please approve

> Important: When implementing volume replication, be sure to use the `Retain` deletionPolicy for the volume snapshot and using the latest import procedure (version 1.7.0 or later) (see Importing an existing volume).
> For more information, see the [Volume Snapshot Classes > Deletion Policy](https://kubernetes.io/docs/concepts/storage/volume-snapshot-classes/#deletionpolicy) information in the Kubernetes documentation.

Reply:
sorry for the misleading link, I meant to put [this link](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/), snapshots are not relevant here. I confused deletion with reclaim

Reply:
for replication to work, the PV of the replication source must have the internal id in its volumeHandle.
the only way I know to "change" a PV volumeHandle is to delete it using "retain" reclaim policy, and import it with a volumeHandle that has said internal id

Reply:
Important: When implementing volume replication, be sure to use the Retain reclaim policy for the PersistentVolume and using the latest import procedure (version 1.7.0 or later) (see Importing an existing volume).
For more information, see the [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) information in the Kubernetes documentation.

Is this sufficient? I didn't understand the last comment that you made.

> for replication to work, the PV of the replication source must have the internal id in its volumeHandle.
> the only way I know to "change" a PV volumeHandle is to delete it using "retain" reclaim policy, and import it with a volumeHandle that has said internal id

If you need some of this added in, please suggest how to word the limitation and I'll rework as necessary. thanks

Reply:
@ArbelNathan  Please comment regarding the adding of vdisk information to the import volume.
https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r714803798

I need specific information as to what/how to add this in. i understand that the general name is "internal ID" but that doesn't help with finding it for A9K or how to do it via GUI for either the SVC or the A9K.
please give the specific information that you want me to write (I'll fix up as needed). thank you

Reply:
as I wrote, I don't think this belongs here, it is a replication-only backward incompatibility issue.

> Is this sufficient? I didn't understand the last comment that you made.
no, this implies all volumes were created with a driver version lower than 1.7.0, but as of 1.7.0 the driver creates PVs that are ready for replication.
in addition, I'm not sure it's clear that "use the Retain reclaim policy" means "delete the PV using `Retain` reclaim policy".
I would also change `and using the latest import` to `and import it using the latest import`.
IMO it is also not completely clear that the relevant PV is the replication datasource.

> If you need some of this added in, please suggest how to word the limitation and I'll rework as necessary. thanks
no, I just want to make sure you have the full picture

Reply:
regarding the value of the parameter, its the id of the volume object, no matter how we will get it, its the `id` field. (no `vdisk_uid` nor `wwn`, just `id`)

Reply:
@ArbelNathan 
1. The [svc](https://www.ibm.com/docs/en/flashsystem-9x00/8.4.x?topic=commands-lsvdisk) `lsvdisk` command shows ID under the same filtervalue? that is used to find the vdisk_UID. Is this the correct one that they should be using?
2. What volume property value should they be using for the SV GUI?
3. is this the same "system id" that is mentioned in the important note? please see how i've reworded some of the sections and approve.

Replication is currently only for SVC so we'll do those only. 

> 2. Take the volume_name and other optional information (collected before the procedure) and insert it into the YAML file (under spec.csi.volumeAttributes).
> 
> **Important**: If using the CSI Topology feature, the spec.csi.volumeHandle contains the system ID. In the example below, the spec.csi.volumeHandle would read similar to the following: `SVC:demo-system-id-1:600507640082000B08000000000004FF`.
> 
> 

@oriyarde  the important note with the limitation was input into limitations section. let's continue regarding this specific issue there.

Reply:
thanks, [I'm continuing there](https://github.com/IBM/ibm-block-csi-driver/pull/360/files#r716603820). I will let Arbel continue here

Reply:
LGTM
@oriyarde I believe you can resolve